interfaces/apparmor/spec.go

```diff
@@ -736,7 +737,7 @@ func emitEnsureDir(spec *Specification, ifaceName string, ensureDirSpec *interfa
 		return
 	}
 	for iter.Next() {
-		if iter.CurrentPathNoSlash() == mustExistDir {
+		if iter.CurrentPath() == mustExistDir {
```

Comment:
This is correct because thew new `CurrentPath` is exactly like `CurrentPathNoSlash`